### DISCUSSION TITLE
MTA User Control Panel

### DISCUSSION THREAD

**Discussion by saviorsoldier** [OwlGamingCommunity/MTA#9] [General]
I don't find anything related to release of UCP. Is MTA UCP going to be open-source?

**Comment by ChaosParadise** [comment]
It is not at this time no.

**Reply by djawi87** [reply]
hello chaos how about documentation script features about this server i cant access mta thread on owl forums that say didnt have permissions. may i can get all documentation about script from beggining

**Comment by BordoSlx06** [comment]
Hello, will the UCP package be shared?

**Comment by n3rdXXX** [comment]
Hi would the ucp be shared or still not possible atm
